### PR TITLE
Refactor people

### PR DESCRIPTION
**Pull request by akmiller01**
Refactor people page to allow for flowing page layout like this:

image

### CODE REVIEW

File: iati/about/models.py [hunk 1]
```diff
@@ -1,15 +1,15 @@
 from django.db import models
 
 from wagtail.admin.edit_handlers import InlinePanel
-from wagtail.core.blocks import CharBlock, StreamBlock, StructBlock, TextBlock
+from wagtail.core.blocks import CharBlock, RichTextBlock, StreamBlock, StructBlock, TextBlock
```
Comment (dalepotter) [inline iati/about/models.py]: Not sure `RichTextBlock` is used on this page, so can be safely removed :-)

**Comment by akmiller01** [comment]
Merge conflicts solved, does @allthatilk still want to review?

**Comment by dalepotter** [comment]
Looks good, but... think I might have found a bug.

With this data in the CMS:

image

The description doesn't seem to show.

image

Also there it seems to default to the Oxfam logo when no org logo is added.

**Comment by akmiller01** [comment]
@dalepotter  fixed the logo and the description text. I guess I shouldn't have assumed that "logo.png" was an IATI logo!

**Comment by dalepotter** [comment]
yay - the description and default logos work now...

Just tried entering in some data and came across a new issue though - when you add a photo for just one of the people, it shows up for both:

**CMS view**
image

**Page view**
image

**Comment by akmiller01** [comment]
Good catches all round! Fixed it by moving the image tag after the boolean check to see if the image existed.

**Comment by allthatilk** [comment]
@akmiller01  is this ready for re-review?

**Comment by akmiller01** [comment]
@allthatilk, this is complete.

**Comment by dalepotter** [comment]
> @allthatilk, this is complete.

Marking as complete

File: README.md [hunk 2]
```diff
@@ -29,12 +29,8 @@ pip install -r requirements_dev.txt
 createdb iati-website
 cp settings/local.py.example settings/local.py
```
Comment (dalepotter) [inline README.md]: Bit confused about how these changes have made it into this branch. Perhaps `add-about-models` has not been updated with `add-page-models`??

File: iati/home/templates/home/includes/sidepanel.html [hunk 3]
```diff
@@ -7,56 +7,55 @@ <h3 class="aside-m__heading">In this section</h3>
     <button class="navigation-secondary-toggle" type="button" id="navigation-secondary-toggle">pages in this section <i role="presentation" aria-hidden="true" class="ico ico--16 ico-plus-white"></i></button>
     <nav class="navigation-secondary" id="navigation-secondary">
         <ul class="navigation-secondary__list">
-        {% if self.depth == 3 %}
```
Comment (dalepotter) [inline iati/home/templates/home/includes/sidepanel.html]: Also not sure how these changes are part of this PR?

**Comment by dalepotter** [comment]
@akmiller01 Changes to the people page look, though before approving it would be good to see if you have any thoughts on how the extra code (in https://github.com/IATI/preview-website/pull/69#pullrequestreview-116836719 and https://github.com/IATI/preview-website/pull/69#pullrequestreview-116837386) have made it into this PR.

**Comment by dalepotter** [comment]
Marking as incomplete, pending a quick chat with @akmiller01 on the changes in this branch that appear to be unrelated to the people page :-)

**Pull request merged**